aten/src/ATen/native/native_functions.yaml

```diff
@@ -58,6 +58,8 @@
 
 - func: unsqueeze_(Tensor self, int64_t dim) -> Tensor
 
+- func: _det_with_svd(Tensor self) -> (Tensor, Tensor, Tensor, Tensor)
```

Comment:
@colesbury What is the naming convention we are suppose to use here? I think this should be something like `det_forward` and we are suppose to also define `det` which only returns the first argument.

Reply:
This seems good to me. We don't really want this as part of the public-API, but it's useful for avoiding duplicate computations in the backwards. The underscore prefix seems appropriate for this.

With the NN functions, we named some `xxx_forward` because we thought that the `_forward` versions might be useful at least at the C++ level. I don't think that's the case here.

As you suggest below, I think moving det() down to C++ as a simple NativeFunction would be good too.